spread.yaml

```diff
@@ -109,6 +109,7 @@ backends:
                   workers: 6
             - ubuntu-18.04-64:
                   workers: 8
+                  storage: 12G
```

Comment:
master has this already afaict I can tell, if you move the line before workers it should merge cleanly without having this twice

Reply:
Moved in d10853b04e